runtime/doc/eval.txt

```diff
@@ -5138,6 +5138,22 @@ msgpackparse({list})				   {Nvim} *msgpackparse()*
 			representing extension type. Second is 
 			|readfile()|-style list of strings.
 
+msgpane({message}[, {highlight}[, {open}]])		{Nvim} *msgpane()*
```

Comment:
I was thinking from the perspective of debug logging for plugin development.  I thought it would be nice if a message could get into the buffer without overwriting what's displayed in the command line.  I also wanted a way to make sure an important message wouldn't go unnoticed.